tests/e2e/psp34/psp34.tests.ts

```diff
@@ -181,7 +181,7 @@ describe('MY_PSP34', () => {
     // Act - Alice transfers the token form sender to bob
     await expect(contract.query.getCallCounter()).to.have.output(0)
     await tx.transfer(contract.address, PSP34Args.IdBuilder.U8(0), 'data' as unknown as string[])
```

Comment:
we can remove safe transfer tests (one test with transfer is enough, also gotta rename test case)